sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/DerivationToolExt.kt

```diff
@@ -20,8 +20,16 @@ fun Derivation.deriveUnifiedAddress(
 fun Derivation.deriveUnifiedSpendingKey(
     seed: ByteArray,
     network: ZcashNetwork,
-    account: Account
-): UnifiedSpendingKey = UnifiedSpendingKey(deriveUnifiedSpendingKey(seed, network.id, account.value))
+    accountIndex: Int
+): UnifiedSpendingKey =
+    UnifiedSpendingKey(
+        JniUnifiedSpendingKey(
+            // FIXME: How to construct JniUnifiedSpendingKey without accountUuid?
+            // FIXME: The tests fixture currently used to pass tests
+            accountUuid = AccountFixture.new().accountUuid,
```

Comment:
This is a bit tricky now. As the `deriveUnifiedSpendingKey` started to return `ByteArray` instead of the previously used `JniUnifiedSpendingKey`. Now we can't easily construct `UnifiedSpendingKey` just from `accountIndex`. Any ideas please? @str4d @nuttycom

Reply:
I see now. I just got details about this from @LukasKorba that we should remove all account-related information from spending and viewing key objects. Then, we can update this easily.